big_tests/tests/connect_SUITE.erl

```diff
@@ -166,6 +170,10 @@ init_per_group(fast_tls,Config)->
     [{tls_module, fast_tls} | Config];
 init_per_group(session_replacement, Config) ->
     Config;
+init_per_group(proxy_protocol, Config) ->
+    ejabberd_node_utils:modify_config_file([{tls_config, "{proxy_protocol, true},"}], Config),
```

Comment:
This is done a few times in this module - I can add a comment there. It could be better to add a template for all extra listener options, but I think this is outside of the scope of this PR.

Reply:
Right, that's outside of the scope of this PR. On the other hand if we don't do it now, or at least don't add it to our backlog it will quickly be forgotten. Do we have other places (other tests SUITEs) where the extra listener options could be used?